Added release notes for Tyk cloud (#5622)
TykTechnologies/tyk-docs: tyk-docs/content/developer-support/release-notes/cloud.md

---
title: Tyk Cloud Release Notes
date: xx
description: "Release notes documenting updates, enhancements, and changes for Tyk Cloud"
tags: ["Tyk Cloud", "Release notes", "v1.23", "1.23.0", "changelog"]
---

## 1.23.0 Release Notes

### Release Date 14 of November 2024

### Release Highlights

> This release adds new features to Tyk Cloud, including Support for **Multiple Plugin Bundles** in MServ for greater flexibility in plugin management, and an **Embedded Product Tour** during deployment wait times, offering users a guided overview of Tyk Dashboard features.

Reviewer: @sharadregoti we need content that explains what mserv is and be able to link to it

Author: We will use this [explanation](tyk-cloud/configuration-options/using-plugins/uploading-bundle/).

> This release adds new features to Tyk Cloud, including Support for **Multiple Plugin Bundles** in MServ for greater flexibility in plugin management, and an **Embedded Product Tour** during deployment wait times, offering users a guided overview of Tyk Dashboard features.

Reviewer: ```suggestion
This release adds new features to Tyk Cloud, including Support for **Multiple plugin bundles** in *MServ* for greater flexibility in plugin management, and an **Embedded Product Tour** during deployment wait times, offering users a guided overview of Tyk Dashboard features.

```

For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-1.23.0">}}) below.

### Breaking Changes
<!-- Required. Use the following statement if there are no breaking changes, or explain if there are -->

There are no breaking changes in this release

### Dependencies {#dependencies-1.23.0}

> Here you can find the link to [Long-Term Support Releases](https://tyk.io/docs/developer-support/special-releases-and-features/long-term-support-releases/)

Author: Removed this section. As it does add any value.

Here it is important to explain the benefit of each changelog item. As mentioned by James in a previous Slack message (https://tyktech.slack.com/archives/C044R3ZTN6L/p1686812207060839?thread_ts=1686762128.651249&cid=C044R3ZTN6L):
"...it is important to document the customer impact for the work delivered, so we can share it with prospects/install base. For example:
"New Chart delivers x and y benefit to a and b customer use cases. The business impact for them will be this and that" -->

#### Added
<!-- This section should be a bullet point list of new features. Explain:
- The purpose of the new feature
- How does the new feature benefit users?
- Link to documentation of the new feature
- For OSS - Link to the corresponding issue if possible on GitHub to allow the users to see further info.
Each change log item should be expandable. The first line summarises the changelog entry. It should be then possible to expand this to reveal further details about the changelog item. This is achieved using HTML as shown in the example below. -->

<ul>
  <li>
    <details>
      <summary>

>         Contact Form for PoC Requests on Trial Expiration

Reviewer: Can we avoid capitalizing every word of the summary? it looks odd on the page

---
title: Tyk Cloud Release Notes
date: xx
description: "Release notes documenting updates, enhancements, and changes for Tyk Cloud"
tags: ["Tyk Cloud", "Release notes", "v1.23", "1.23.0", "changelog"]
---

## 1.23.0 Release Notes

### Release Date 14 of November 2024

### Release Highlights

> This release adds new features to Tyk Cloud, including Support for **Multiple plugin bundles** in *MServ* for greater flexibility in plugin management and an **Embedded product tour** during deployment wait times, offering users a guided overview of Tyk Dashboard features.

Reviewer: ```suggestion
This Tyk Cloud update introduces features that improve both flexibility in plugin management and user onboarding. Now, *Mserv* (a Tyk CLI Tool, used as a file server for plugin definitions),  supports **multiple plugin bundles**, allowing greater customization and easier deployment of plugin configurations. Additionally, we added an **embedded product tour* to enhance the deployment experience, offering a guided walkthrough of Tyk Dashboard’s features, ideal for users familiarizing themselves with the platform during onboarding.

```

> For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-1.23.0">}}) below.

Reviewer: ```suggestion
For a comprehensive list of improvements and fixes, please check the detailed [changelog]({{< ref "#Changelog-1.23.0">}}) below.
```
